src/style.css

```diff
@@ -2,0 +1,107 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.page {
+  margin: 0;
+  padding: 0;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bar {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding-top: 20px;
+}
+
+.search-bar__item {
+  display: flex;
+  align-items: center;
+
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+}
+
+.search-bar__item--big {
+  height: 70px;
+  line-height: 70px;
+
+  margin-bottom: 20px;
+  border-radius: 3px;
+}
+
+.search-bar__item--small {
+  height: 42px;
+  line-height: 42px;
+  font-size: 14px;
+
+  border-radius: 4px;
+}
+
+.icon-big {
+  width: 19px;
+  height: 19px;
+  background-image: url(./images/Search.svg);
+  background-size: cover;
+  background-position: center;
+  margin-left: 26px;
+  margin-right: 17px;
+}
+
+.icon-small {
+  width: 11px;
+  height: 11px;
+  background-image: url(./images/Search.svg);
+  background-size: cover;
+  background-position: center;
+  margin-left: 13px;
+  margin-right: 9px;
+}
+
+.input {
+  border: none;
```

Comment:
Avoid setting the border to 'none' as it might affect the overall layout. Instead, you can set the border-color to 'transparent'.